samples/tfm_integration/psa_crypto/src/main.c

```diff
@@ -44,5 +44,7 @@ int main(void)
 	/* Dump any queued log messages, and wait for system events. */
 	al_dump_log();
 
+	LOG_INF("Done.");
+
```

Comment:
Is this change required?

Reply:
To this and your other comment: Not strictly required as in the sample would be broken without them, but those very much help someone looking at the logs to understand that the sample is done instead of having hung. So to me yes, those are important enough to have.